delft/sequenceLabelling/models.py

```diff
@@ -870,10 +870,12 @@ def __init__(self, config, ntags=None, load_pretrained_weights=True, local_path:
         x = Concatenate()([text_embedding_layer, features_embedding_out])
         x = Dropout(config.dropout)(x)
 
+        '''
```

Comment:
Was this intentional?

Reply:
Nope I was testing features with architecture having a transformer layer, because it's not working as expected. 
I am still working on this branch, I should have put a draft flag !